src/pages/calendar/components/AddTask/RadioButton.jsx

```diff
@@ -0,0 +1,74 @@
+import { useTranslation } from 'react-i18next'
+import styled, { css } from 'styled-components'
+
+export const RadioButton = ({ title, checked, onChange }) => {
+  const { t } = useTranslation();
+  let radioColor = '';
+  let priority=title.charAt(0).toUpperCase() + title.slice(1);
+
+  switch (title) {
```

Comment:
"title" looks not so good.

title means some string which are showing as title
but:
1. you are using it's like a types of button
2. you are using it like value for radio button
3. you are using it like actually title
4. you are setting it like id of the radio button

Depends on it you will have a lot of bugs

Reply:
+